src/problem_bank_scripts/tui/tui.py

```diff
@@ -99,7 +110,7 @@ def other_asks(part: dict, solution: str, use_gpt: bool, exercise: dict | None =
         case "multiple-choice" | "dropdown" | "checkbox":
             options: list[str] = []
             # answer = questionary.text("Solution").ask()
-            num_options = ask_int("Number of options (excluding solution)", default=3)
+            num_options = ask_int("Number of options", default=4)
```

Comment:
Oh okay, I changed this because I got the same number of multiple choice options in the template as `num_options`, instead of adding one more for the solution, but I know at least one of those times was accidentally pasting the solution in as an option, so maybe the other time was too.

Reply:
I think its just that with this you need to specify the correct answer again exactly, on main (and here), the correct answer is automatically added to the options if it doesn't already exist in this function:
https://github.com/open-resources/problem_bank_scripts/blob/27fbef2a9c76f4632a4f05cc8e00aafce72f5c9a/src/problem_bank_scripts/tui/tui.py#L45-L73